src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    const getPeopleFromServer = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    getPeopleFromServer();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading
+            ? <Loader />
+            : (
+              <>
+                {isError && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    Something went wrong
+                  </p>
+                )}
+
+                {people.length <= 0 && (
+                  <p data-cy="noPeopleMessage">
+                    There are no people on the server
+                  </p>
+                )}
+
+                <PeopleTable people={people} personId={slug} />
```

Comment:
Let's render PeopleTable only if there are no error and this array is not empty
<img width="746" alt="Screenshot 2023-04-19 at 10 15 31" src="https://user-images.githubusercontent.com/84275496/232997692-bfc85168-5bfa-4635-945b-b43030f8e72d.png">